ci/docker/runtime_functions.sh

```diff
@@ -336,6 +336,35 @@ build_ubuntu_cpu_cmake_debug() {
     popd
 }
 
+build_ubuntu_cpu_cmake_asan() {
+    set -ex
+
+    pushd .
+    cd /work/build
+    export CC="gcc-8"
```

Comment:
Maybe add cmake arguments `-DCMAKE_CXX_COMPILER=/usr/bin/g++-8` `-DCMAKE_CC_COMPILER=/usr/bin/gcc-8` instead? It would be more cmake'ish.